assets/js/app.js

```diff
@@ -43,7 +44,7 @@ Hooks.payload = {
     // Channels can be named anything
     // All clients on the same Channel will get messages sent to that Channel
     this.channel = this.realtimeSocket.channel(channelName, {
-      config: { broadcast: { self: true } },
+      config: { broadcast: { self: true, private: private_channel } },
```

Comment:
Pretty sure this comes over as a string so you probably need to convert it to a bool